middleware/instrument.go

```diff
@@ -105,6 +110,17 @@ func (i Instrument) Wrap(next http.Handler) http.Handler {
 			labelValues = append(labelValues, tenantID)
 			instrument.ObserveWithExemplar(r.Context(), i.PerTenantDuration.WithLabelValues(labelValues...), respMetrics.Duration.Seconds())
 		}
+		if i.SlowRequestCutoff > 0 && respMetrics.Duration > i.SlowRequestCutoff {
+			parts := strings.Split(w.Header().Get("Server-Timing"), ", ")
+			volume := int64(0)
+			for _, part := range parts {
+				if strings.HasPrefix(part, i.ServerThroughputUnit) {
+					_, _ = fmt.Sscanf(part, i.ServerThroughputUnit+"=%d", &volume)
```

Comment:
if the format doesn't comply, what do you think about not observing the throughput at all? I mean if upstream didn't properly communicate what volume it processed, then how can we measure the throughput?